README.md

```diff
@@ -1,6 +1,6 @@
 # [pyp-w2] Collections Hierarchy
 
-Today, we will re implement part of the Python collections hierarchy using Mixins.
+Today, we are going to implement part of the Python collections hierarchy using Mixins.
```

Comment:
Ohh, this makes sense!

On Sat, Jul 23, 2016 at 2:29 AM, corez92 <notifications@github.com> wrote:

> In README.md
> <https://github.com/rmotr-group-projects/pyp-w2-gw-collections-hierarchy/pull/21#discussion_r71945230>
> :
>
> > @@ -1,6 +1,6 @@
> >  # [pyp-w2] Collections Hierarchy
> >
> > -Today, we will re implement part of the Python collections hierarchy using Mixins.
> > +Today, we are going to implement part of the Python collections hierarchy using Mixins.
>
> I thought it was meant to be re-implement since we are just making new
> list and dict objects, so implementing new objects
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rmotr-group-projects/pyp-w2-gw-collections-hierarchy/pull/21/files/c3c1921d8179abdda6051c1d0fd76f0637aad4dd#r71945230>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAzPEtX5GPxX6yGt8cVm8-AIA_9ohy4nks5qYS8pgaJpZM4JTB_e>
> .
>